x-pack/filebeat/input/netflow/testdata/golden/IPFIX-Barracuda-extended-uniflow-template-256.golden.json

```diff
@@ -0,0 +1,153 @@
+{
+  "test_name": "IPFIX Barracuda extended uniflow template 256",
+  "events": [
+    {
+      "Timestamp": "2018-04-18T08:16:47Z",
+      "Meta": null,
+      "Fields": {
+        "destination": {
+          "ip": "64.235.151.76",
+          "locality": "public",
+          "port": 443
+        },
+        "device": {
```

Comment:
`device` is now called `observer`. This would be data about the tool monitoring the network traffic, correct?

Reply:
Yes, will change.